src/libs/sys-util-rs/src/fs.rs

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) 2019-2021 Alibaba Cloud
+// Copyright (c) 2019-2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::ffi::OsString;
+use std::fs::{self, File};
+use std::io::{Error, Result};
+use std::os::unix::io::AsRawFd;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+use crate::{eother, sl};
+
+const FUSE_SUPER_MAGIC: u32 = 0x65735546;
```

Comment:
Might be good to add a reference to the kernel source that defines this.

Reply:
done